### PR TITLE
Bugfix/2277 fix solutions management

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: scripts, solutions

**Context**: 

**Summary**:

**Acceptance criteria**: 

---

See: #2277

### CODE REVIEW

**Review by TeddyAndrieux — APPROVED** [review]
LGTM

File: scripts/solutions.sh [hunk 1]
```diff
@@ -261,6 +259,11 @@ configure_archives() {
     local removed=${1:-False}
 
     for archive in "${ARCHIVES[@]}"; do
+        if file "$archive" | grep -vq 'ISO 9660'; then
```
Comment (Ebaneck) [inline scripts/solutions.sh:262]: Minor and can be omitted
We are searching for the absence of a  fixed string here `ISO 9660` so we can make this faster with `fgrep`?? no??

**Review by Ebaneck — APPROVED** [review]

File: scripts/solutions.sh (hunk 1 above)
Comment (slaperche-scality) [reply scripts/solutions.sh:262]: Or `grep -F` (`fgrep` is deprecated, as explained in the man page).

But I don't think we will observe a perf improvement here ^^

**Pull request merged**